src/index.js

```diff
@@ -36,4 +37,20 @@ class TelegramLoginButton extends React.Component {
   }
 }
 
+TelegramLoginButton.propTypes = {
+  botName: PropTypes.string.isRequired,
+  dataOnauth: PropTypes.func,
+  dataSize: PropTypes.oneOf(['large', 'medium', 'small']),
+  requestAccess: PropTypes.string,
+  usePic: PropTypes.bool,
+};
+
+TelegramLoginButton.defaultProps = {
+  botName: 'samplebot',
+  dataOnauth: () => undefined,
+  dataSize: 'large',
+  requestAccess: 'write',
```

Comment:
We only need define default props for props that `isRequired` right?

Reply:
No, for those who are **not** required. But nothing stops the user from not entering required props, so I'd often define default props for default ones too.